test/EndToEnd/ProjectTemplates/UwpPackageRefClassLibrary.zip/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
```

Comment:
Can this file be removed? I don't think it would contribute anything to the test so it would be nice to reduce the extra stuff in the repo.